compat/src/main/scala-2.11_2.12/scala/collection/compat/BuildFrom.scala

```diff
@@ -41,7 +39,7 @@ object BuildFrom {
   implicit def fromCanBuildFrom[From, A, C](
       implicit cbf: CanBuildFrom[From, A, C]): BuildFrom[From, A, C] =
     new BuildFrom[From, A, C] {
-      def fromSpecificIterable(from: From)(it: Iterable[A]): C = (cbf(from) ++= it).result()
+      def fromSpecific(from: From)(it: IterableOnce[A]): C = (cbf(from) ++= it).result()
```

Comment:
Are there any test coverage for this change?

Reply:
I've added a test to ensure that a `BuildFrom` implementation compiles on all versions.